tests/v2/validation/pipeline/Jenkinsfile.multibranch.recurring

```diff
@@ -0,0 +1,129 @@
+#!groovy
+node {
+    def rootPath = "/root/go/src/github.com/rancher/rancher/"
+    def workPath = "/root/go/src/github.com/rancher/rancher/tests/v2/validation/"
+    def rancherImageTags = "${env.RANCHER_IMAGE_TAGS}".split(',')
+    def jobName = "${JOB_NAME}"
+    def config = env.CONFIG
+    def filename = "config.yaml"
+    def hostnameSuffix = ["v2.7-head":"v27", "v2.8-head":"v28", "v2.9-head":"v29"]
+    def releaseBranchTagMap = ["v2.7-head":"release/v2.7", "v2.8-head":"release/v2.8", "v2.9-head":"release/v2.9"]
+    def tagChartMap = ["v2.7-head":"2.7.12", "v2.8-head":"2.8.3", "v2.9-head":"2.8.3"]
+    def rancherChartRepo = "prime"
+    def rancherChartUrl = "https://charts.rancher.com/server-charts"
+    def rancherImage = "stgregistry.suse.com/rancher/rancher"
+    def cattleImageEnvVar = '[\\"CATTLE_AGENT_IMAGE|stgregistry.suse.com/rancher/rancher-agent:v2.7-head\\"]'
+    def configPath = ""
+
+    if (jobName.contains('/')) { 
+      jobNames = jobName.split('/')
+      jobName = jobNames[jobNames.size() - 1] 
+    }
+
+    def branch = "release/v2.9"
```

Comment:
not blocking approval, but do we usually leave 1 default branch in the jenkinsfile across the rancher branches? i.e. would this be 2.8 in this PR instead?

Reply:
This part doesn't matter, tbh. I just back ported everything for parity sake, we really don't need them.